setup.sh

```diff
@@ -47,9 +47,9 @@ install-mailscanner
 
 ##ask directory for web files
 logprint ""
-ask "In what location should MailWatch be installed (web files directory)?[/var/www/html/mailscanner/]:" WebFolder
+ask "In what location should MailWatch be installed (web files directory)?[/opt/mailwatch/]:" WebFolder
 if [ -z $WebFolder ]; then
-    WebFolder="/var/www/html/mailscanner/"
+    WebFolder="/opt/mailwatch/"
```

Comment:
Your right. 
@endelwar did you already decided on the new locations under /opt/mailwatch?